README.Rmd

```diff
@@ -2,41 +2,43 @@
 output: github_document
 ---
 
-<!-- README.md is generated from README.Rmd. Please edit README.Rmd -->
+<!-- README.md is generated from README.Rmd. Please edit that file -->
 
-```{r, echo = FALSE, warning=FALSE, message=FALSE}
+```{r}
+#| include: false
 knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>",
-  fig.path = "man/figures/",
-  fig.height = 1
+  fig.path = "man/figures/README-",
+  out.width = "100%"
 )
-require(ggplot2)
 ```
 
 # nmfspalette <a href={https:://nmfs-ost.github.io/nmfspalette}><img src="man/figures/logo.png" alt="logo with an image of a fish in multiple palette colors" align="right" width="139" height="139"/>
 
-An R color palette for `{ggplot2}` using NOAA Fisheries branding colors
```

Comment:
Hi Nick, I think this line was deleted in error?

Reply:
Ah, it's a deletion here, but it is actually just moved to below the badges (line 25), so that the package README goes:

1. pkg name (header)
2. badges
3. overview

Rather than:

1. pkg name (header)
2. overview
3. badges

Reply:
Ah, sorry, I missed that! Please disregard!

Reply:
All good! I was also alarmed when I saw this and wanted to make sure I didn't delete anything important!